### DISCUSSION TITLE
Frontend version of the Calendar

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/PublishPress-Planner#1280] [Feature Requests]
This isn't the first time I've heard this request:

> I want somewhere clients could look at the content planner without having access to the backend.

**Comment by stevejburge** [comment]
One more:

> Can I use shortcode of calendar to display in frontend views? (My thinking is I have a news/blog site where contributors/authors write blogs and will be on review which show on publishpress calendar. Some of our Editors will edit that blogs in frontend and may publish it. I DO NOT WANT THE EDITORS TO VIEW ADMIN PANEL/DASHBOARD).

**Comment by james95128** [comment]
We have a similar requirement but it does not involve frontend editing but simply for anonymous visitors to our site to see published posts. These visitors are students who return to our site to view content that is released on a drip schedule. FYI the content is WordPress posts that display a PDF. We do this capability in English and Spanish.

**Comment by stevejburge** [comment]
@james95128 Can you explain that? Are you looking for a Calendar view on the frontend of your site that is read-only?

**Comment by james95128** [comment]
Yes, correct. Read-only Calendar view.

**Comment by stevejburge** [comment]
One more:

> we would like to show our visitors a calendar that shows scheduled posts to come.
> 
> So far we haven’t been able all plugins are either Event calendars which requiere us to create events for every post (double work ) or editorial calendars without a front end option.
> 
> Is there any way to achieve that with publishpress?
> 
> we found some really old widget plugins that allowed for it, but not in calendar format

**Comment by stevejburge** [comment]
One more:

> It would be an enormously useful feature as many of us run small publications that employ outside help for advertising/marketing/social media promotion, contractors who really need to know what's ahead on the editorial calendar but don't need back-end access to our website...we've tried a number of workarounds but haven't found a good, glitch-free and simple way to do this.

**Comment by rizaardiyanto1412** [comment]
ONe more:

> Does this tool provide any mechanisms to show planned content on the frontend of a WP site as it lopoks predominantly for backend use?
> I.e.
> our client want a tool to show readers whats coming up ate as well as be able to use internally?
